.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
 
 script:
   # Build the WASM binding
+  - npm install --prefix lib/binding_web
```

Comment:
`script/build-wasm` requires rollup to be present so we need to run this first, I suspect some docs also need to be updated.

Reply:
we could also have `script/build-wasm` run `npm install` but that seems somewhat annoying